pkg/utils/addons/addons_test.go

```diff
@@ -0,0 +1,160 @@
+package addons
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	mf "github.com/manifestival/manifestival"
+	"gotest.tools/golden"
+)
+
+func TestCreatePipeline(t *testing.T) {
+	t.Run("dotnet pipeline", func(t *testing.T) {
+		var runtime = map[string]spec{
+			"s2i-dotnet-3": {runtime: "dotnet", minorVersion: "1", supportedVersion: "[3.1,3.0]"},
+		}
```

Comment:
can we make the three tests in this file into a table based test

```
runtime[] = {
map[string]spec{
			"s2i-dotnet-3": {runtime: "dotnet", minorVersion: "1", supportedVersion: "[3.1,3.0]"},
		},
map[string]spec{
			"s2i-nodejs": {runtime: "nodejs", version: "10", supportedVersion: "[10,12]"},
		}
...
}
```

then 
```
for runtime in runtimes
    run tests
```

Reply:
With new implementations, this is completely changed. Thank you for the suggestion.